__tests__/helpers/buildFilterQuery.test.js

```diff
@@ -0,0 +1,43 @@
+const { buildFilterQuery } = require("../../helpers/buildFilterQuery");
+
+describe("buildFilterQuery function", () => {
+  it("should return query for filtering, when we passed query with one param", () => {
+    const fakedQuery = {
```

Comment:
Not necessary to name it `fakedQuery`,  `query` is fine

Reply:
already fixed